Orange/widgets/utils/plot/owplotgui.py

```diff
@@ -31,12 +31,12 @@
 from Orange.widgets import gui
 from Orange.widgets.utils.itemmodels import DomainModel
 
-from .owconstants import *
```

Comment:
Have you checked that no other code relies on having those constants in `owplotgui`'s namespace? Just for the case...

Reply:
Everything needed by `ScatterPlotGraph` is imported. **Parallel Coordinates** may need some other constants but anyway that widget is a dead code. However I will revert this code back just in case.

Reply:
If nobody who currently imports owplotgui uses these names, it's better to remove the import. I just asked whether you checked. If confirmed, please re-remove it.

Reply:
OK, I reverted it back and also did some style changes. That is why I now imported even more constants, now they are used instead of numbers (see https://github.com/biolab/orange3/pull/2541/files#diff-665cab46a232703eb5e7bb8b696303ecR105 ).